packages/katello/rubygem-pulp_container_client/rubygem-pulp_container_client.spec

```diff
@@ -1,100 +1,58 @@
-
+# template: default
 %global gem_name pulp_container_client
 
-%global release 1
-# %%global prerelease rc1
-# %%global prereleaserpm %{?prerelease:.}%{?prerelease}
-
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 2.10.3
-
-Release: %{?prereleaserpm:0.}%{release}%{?prereleaserpm}%{?dist}
-Summary: Pulp container plugin for Pulp3 API Ruby Gem
-Group: Development/Languages
-License: GPLv2+
-URL: https://github.com/pulp/pulp_container/
-Source0: https://rubygems.org/gems/%{gem_name}-%{version}%{?prerelease}.gem
-
-Obsoletes: %{?scl_prefix_ruby}rubygem-pulp_docker_client
+Name: rubygem-%{gem_name}
+Version: 2.10.4
+Release: 1%{?dist}
+Summary: Pulp 3 API Ruby Gem
+License: GPL-2.0+
+URL: https://openapi-generator.tech
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby >= 1.9
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(faraday) >= 0.17
-Requires: %{?scl_prefix}rubygem(faraday) < 1
-Requires: %{?scl_prefix}rubygem(faraday) < 1.9.0
-Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1.0
-Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1
-Requires: %{?scl_prefix_ruby}rubygem(json) < 3
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby >= 1.9
+BuildRequires: ruby >= 1.9
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
 %description
-Autogenerated bindings for the pulp_container plugin for pulp3
+Fetch, Upload, Organize, and Distribute Software Packages.
+
 
 %package doc
-Summary: Documentation for %{pkg_name}
-Group: Documentation
-Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Summary: Documentation for %{name}
+Requires: %{name} = %{version}-%{release}
 BuildArch: noarch
 
 %description doc
-Documentation for %{pkg_name}.
+Documentation for %{name}.
 
 %prep
-%{?scl:scl enable %{scl} - << \EOF}
-gem unpack %{SOURCE0}
-%{?scl:EOF}
-
-%setup -q -D -T -n  %{gem_name}-%{version}%{?prerelease}
-
-%{?scl:scl enable %{scl} - << \EOF}
-gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
-%{?scl:EOF}
+%setup -q -n  %{gem_name}-%{version}
 
 %build
 # Create the gem as gem install only works on a gem file
-%{?scl:scl enable %{scl} - << \EOF}
-gem build %{gem_name}.gemspec
-%{?scl:EOF}
+gem build ../%{gem_name}-%{version}.gemspec
 
 # %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
 # by default, so that we can move it into the buildroot in %%install
-%{?scl:scl enable %{scl} - << \EOF}
-%gem_install -n %{SOURCE0}
-%{?scl:EOF}
+%gem_install
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
 cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
+
+
 %files
 %dir %{gem_instdir}
-%exclude %{gem_instdir}/git_push.sh
+%{gem_instdir}/git_push.sh
```

Comment:
This is something we should exclude. Ideally also make sure it's removed upstream so we don't need to do this every time.